docs/GetStarted/generic_vulkan_env_setup.md

```diff
@@ -95,6 +95,9 @@ $ LD_LIBRARY_PATH={PATH_TO_VULKAN_SDK}/x86_64/lib/
 $ LD_PRELOAD=libvulkan.so.1
 ```
 
+This can also be done by sourcing the proper `setup-env.sh` from one of the
```

Comment:
FWIW, I had been consider sourcing `setup-env.sh` to be part of "installing" the VulkanSDK on Linux. If you just download it and put it in an arbitrary location, it won't affect your system.